synapse/config/federation.py

```diff
@@ -46,6 +75,11 @@ def read_config(self, config, **kwargs):
         # Always blacklist 0.0.0.0, ::
         self.ip_range_blacklist.update(["0.0.0.0", "::"])
 
+        try:
+            self.ip_range_whitelist = IPSet(config.get("ip_range_whitelist", ()))
+        except Exception as e:
+            raise ConfigError("Invalid range(s) provided in ip_range_whitelist: %s" % e)
```

Comment:
as of #8874, I think this will look better if you do:

```suggestion
        except Exception as e:
            raise ConfigError("Invalid range(s) provided in ip_range_whitelist") from e
```

Reply:
maybe update the other instances here too.